src/pytest_cpp/google.py

```diff
@@ -98,7 +98,7 @@ def run_test(self, executable, test_id, test_args=(), harness=None):
                 if failures:
                     return [GoogleTestFailure(x) for x in failures], output
                 elif skipped:
-                    pytest.skip()
+                    [pytest.skip(x) for x in skipped]
```

Comment:
This will really only show the first failure, as `pytest.skip()` raises an exception. Is that what you expect?

Reply:
@nicoddemus Yes, since the tests are run one by one,  that array should only contain one element. So, we are okay 😄

Reply:
Oh I know that, just pointing out that even if a test emitted (say) 3 messages, we only show one skipped message.

We could decide to use this instead:

```python
pytest.skip("\n".join(skipped))
```

Reply:
Went ahead and applied that 👍

Reply:
Awesome @nicoddemus, thanks 😄